Assets/Scripts/ItemsScripts/Item.cs

```diff
@@ -2,7 +2,32 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public interface Item
+public class Item : MonoBehaviour
```

Comment:
what if we need item mot for player stat changing? For example, score, which is handled somewhere else.